{
  "uid" : "f34c95ca497cb18e",
  "name" : "test_001",
  "fullName" : "report_demo.cases.test_result.TestResult#test_001",
  "historyId" : "7c158480cb293e9c8f9bb81fd4fb05fb",
  "time" : {
    "start" : 1617501462577,
    "stop" : 1617501462578,
    "duration" : 1
  },
  "description" : "\n         获取测试用例执行结果: 调用001\n        ",
  "descriptionHtml" : "<pre><code>     获取测试用例执行结果: 调用001\n</code></pre>\n",
  "status" : "failed",
  "statusMessage" : "AssertionError: 实际值: 1 != 预期值: 0",
  "statusTrace" : "self = <report_demo.cases.test_result.TestResult object at 0x000002439E9A89B0>\nsomething = None\n\n>   ???\n\nD:\\BaiduNetdiskDownload\\allure_demo\\report_demo\\cases\\test_result.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ncls = <class 'report_demo.utils.assert_handle.Assertion'>, reality_value = 1\nexpected_value = 0\n\n    @classmethod\n    def assert_equal_value(cls, reality_value, expected_value):\n        \"\"\"\n        校验等于预期值\n        :param reality_value: 实际值\n        :param expected_value: 预期值\n        :return:\n        \"\"\"\n>       assert expected_value == reality_value, \"实际值: {} != 预期值: {}\".format(reality_value, expected_value)\nE       AssertionError: 实际值: 1 != 预期值: 0\n\nutils\\assert_handle.py:19: AssertionError",
  "flaky" : true,
  "beforeStages" : [ {
    "name" : "fix_module",
    "time" : {
      "start" : 1617501461074,
      "stop" : 1617501461427,
      "duration" : 353
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "登录",
      "time" : {
        "start" : 1617501461074,
        "stop" : 1617501461419,
        "duration" : 345
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "mobile",
        "value" : "17802156123"
      }, {
        "name" : "password",
        "value" : "123456"
      }, {
        "name" : "env_url",
        "value" : "'http://www.niuinfo.com'"
      } ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : true,
    "stepsCount" : 1
  }, {
    "name" : "something",
    "time" : {
      "start" : 1617501462576,
      "stop" : 1617501462576,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "testStage" : {
    "description" : "\n         获取测试用例执行结果: 调用001\n        ",
    "status" : "failed",
    "statusMessage" : "AssertionError: 实际值: 1 != 预期值: 0",
    "statusTrace" : "self = <report_demo.cases.test_result.TestResult object at 0x000002439E9A89B0>\nsomething = None\n\n>   ???\n\nD:\\BaiduNetdiskDownload\\allure_demo\\report_demo\\cases\\test_result.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ncls = <class 'report_demo.utils.assert_handle.Assertion'>, reality_value = 1\nexpected_value = 0\n\n    @classmethod\n    def assert_equal_value(cls, reality_value, expected_value):\n        \"\"\"\n        校验等于预期值\n        :param reality_value: 实际值\n        :param expected_value: 预期值\n        :return:\n        \"\"\"\n>       assert expected_value == reality_value, \"实际值: {} != 预期值: {}\".format(reality_value, expected_value)\nE       AssertionError: 实际值: 1 != 预期值: 0\n\nutils\\assert_handle.py:19: AssertionError",
    "steps" : [ {
      "name" : "test_001",
      "time" : {
        "start" : 1617501462577,
        "stop" : 1617501462577,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: 实际值: 1 != 预期值: 0\n",
      "statusTrace" : "  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"D:\\BaiduNetdiskDownload\\allure_demo\\report_demo\\cases\\test_result.py\", line 21, in test_001\n  File \"D:\\allure_demo\\report_demo\\utils\\assert_handle.py\", line 19, in assert_equal_value\n    assert expected_value == reality_value, \"实际值: {} != 预期值: {}\".format(reality_value, expected_value)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "something",
        "value" : "None"
      } ],
      "shouldDisplayMessage" : true,
      "attachmentsCount" : 0,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 0,
    "hasContent" : true,
    "stepsCount" : 1
  },
  "afterStages" : [ {
    "name" : "fix_module::0",
    "time" : {
      "start" : 1617501462597,
      "stop" : 1617501462733,
      "duration" : 136
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "something::0",
    "time" : {
      "start" : 1617501462588,
      "stop" : 1617501462588,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "story",
    "value" : "get_result_001"
  }, {
    "name" : "feature",
    "value" : "get_result_module"
  }, {
    "name" : "tag",
    "value" : "get_result"
  }, {
    "name" : "parentSuite",
    "value" : "report_demo.cases"
  }, {
    "name" : "suite",
    "value" : "test_result"
  }, {
    "name" : "subSuite",
    "value" : "TestResult"
  }, {
    "name" : "host",
    "value" : "DESKTOP-U6DG9EK"
  }, {
    "name" : "thread",
    "value" : "10804-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "report_demo.cases.test_result"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "237cd686945a9a40",
      "status" : "failed",
      "statusDetails" : "AssertionError: 实际值: 1 != 预期值: 0",
      "time" : {
        "start" : 1617501320309,
        "stop" : 1617501320310,
        "duration" : 1
      }
    }, {
      "uid" : "cb05da03b09c7a54",
      "status" : "failed",
      "statusDetails" : "AssertionError: 实际值: 1 != 预期值: 0",
      "time" : {
        "start" : 1617501266741,
        "stop" : 1617501266742,
        "duration" : 1
      }
    }, {
      "uid" : "1349f24515fe5bb5",
      "status" : "failed",
      "statusDetails" : "AssertionError: 实际值: 1 != 预期值: 0",
      "time" : {
        "start" : 1617501209516,
        "stop" : 1617501209517,
        "duration" : 1
      }
    }, {
      "uid" : "b5e51751ad74a89c",
      "status" : "broken",
      "statusDetails" : "AttributeError: 'SingleInstance' object has no attribute 'assert_equal_value'",
      "time" : {
        "start" : 1617500675103,
        "stop" : 1617500675104,
        "duration" : 1
      }
    }, {
      "uid" : "be9689c7535a11bf",
      "status" : "broken",
      "statusDetails" : "AttributeError: 'SingleInstance' object has no attribute 'assert_equal_value'",
      "time" : {
        "start" : 1617500579136,
        "stop" : 1617500579136,
        "duration" : 0
      }
    }, {
      "uid" : "c1f624c3e468207c",
      "status" : "broken",
      "statusDetails" : "AttributeError: 'SingleInstance' object has no attribute 'assert_equal_value'",
      "time" : {
        "start" : 1617500524892,
        "stop" : 1617500524892,
        "duration" : 0
      }
    }, {
      "uid" : "cec78a129752b5e7",
      "status" : "broken",
      "statusDetails" : "requests.exceptions.ConnectionError: HTTPConnectionPool(host='www.base.com', port=80): Max retries exceeded with url: /test/user/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002497D32A630>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))",
      "time" : {
        "start" : 1617500465349,
        "stop" : 1617500465349,
        "duration" : 0
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "get_result" ]
  },
  "source" : "f34c95ca497cb18e.json",
  "parameterValues" : [ ]
}